src/app/projects/components/projects/projects.component.ts

```diff
@@ -227,12 +228,20 @@ export class ProjectsComponent implements OnInit, OnDestroy {
     if (value == 'Further sample projects') {
       window.open("https://github.com/code-kern-ai/refinery-sample-projects", "_blank");
     } else {
-      this.importSampleProject(this.isProjectInitial ? this.previousValue + ' - initial' : value)
+      this.projectName = this.isProjectInitial ? this.previousValue + ' - initial' : value;
+      this.projectType = this.isProjectInitial ? this.previousValue + ' - initial' : value;
```

Comment:
whats the difference if they are exactly the same?

Reply:
This is for the first creation of the project (the name and the type are the same), but I can just assign the value, no need to calculate the values again.